api/services/control/control.proto

```diff
@@ -92,8 +101,15 @@ message CacheOptionsEntry {
 	map<string, string> Attrs = 2;
 }
 
+message ExporterResponse {
+	map<string, string> Response = 1;
+	// Name identifies the exporter. It is empty
+	// for the common exporter metadata (e.g. cache exporter)
+	string Name = 2;
+}
+
 message SolveResponse {
-	map<string, string> ExporterResponse = 1;
+	ExporterResponse ExporterResponse = 1;
```

Comment:
Is this backward compatible in proto level?

Reply:
Even if it is, I'll go back to the previous message layout - there is no more need for this.